salt/modules/vsphere.py

```diff
@@ -218,6 +238,30 @@
 except ImportError:
     HAS_PYVMOMI = False
 
+# vSphere SDK Automation
+try:
+    # pylint: disable=unused-import
```

Comment:
Should be:
```python
# pylint: disable=unused-import
... a bunch of imports
# pylint: enable=unused-import
```
Or
```python
import blah  # pylint: disable=unused-import
```

Ie, inline disabled are ok, but if the disable is on its own line, you've disabled the rule for the rest of the file. Which is ok if that's the purpose, if not, you then need to enable the rule again.

Reply:
Understood, I'll drop a change for that by end of day. Thanks for clarifying!

Reply:
Should be updated now, if there's no other changes I'll get this asked to be merged/ 👍